src/ViewItem.js

```diff
@@ -291,15 +325,7 @@ class ViewItem extends React.Component {
             lastMenu={detailMenu}
             dismissible
             onClose={this.props.onCloseViewItem}
-            actionMenuItems={[{
-              label: formatMsg({ id: 'ui-inventory.editItem' }),
-              href: this.craftLayerUrl('editItem'),
-              onClick: this.onClickEditItem,
-            }, {
-              id: 'clickable-copy-item',
-              onClick: () => this.onCopy(item),
-              label: formatMsg({ id: 'ui-inventory.copyItem' })
-            }]}
+            actionMenu={actionMenuProps => this.getActionMenu(actionMenuProps, item)}
```

Comment:
Same as was mentioned above in `ViewInstance.js`

Reply:
I will move the logic into the `getActionMenu`-method. 👍